vllm/compilation/backends.py

```diff
@@ -485,7 +487,8 @@ def __call__(self, graph: fx.GraphModule, example_inputs) -> Callable:
             factors = []
             # 0. factors come from the env, for example, The values of
             # VLLM_PP_LAYER_PARTITION will affect the computation graph.
-            env_hash = envs.compute_hash()
+            from vllm.config.utils import hash_factors
+            env_hash = hash_factors(envs.compile_factors())
```

Comment:
This way you can reuse `env_factors` below?
```suggestion
            env_factors = envs.compile_factors()
            env_hash = hash_factors(env_factors)
```